storage/storage_test.go

```diff
@@ -30,14 +30,12 @@ import (
 	ddigest "github.com/opencontainers/go-digest"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/require"
 )
 
 var (
-	_imgd      types.ImageDestination = &storageImageDestination{}
-	_imgs      types.ImageSource      = &storageImageSource{}
-	_ref       types.ImageReference   = &storageReference{}
-	_transport types.ImageTransport   = &storageTransport{}
-	topwd                             = ""
+	// golangci-lint these variables are unused
```

Comment:
I have no idea what this comment is trying to say. The variable _is_ used.

Reply:
This belongs into the commit message not at the place were variables have been removed.